specs/FamilySafety.md

```diff
@@ -0,0 +1,114 @@
+Safety feature is enabled or disabled for a browser process instance, and the end developer 
+won't be able to change it while the webview2 browser process instance is running. Once enabled, 
+it will provide the same functionally as the browser Family Safety feature including: Activity 
+reports, search filtering, and web filtering. Besides toggling the feature, we also provide an 
+API to add desired sites to the allowed uris, so app functionality will not be impacted by 
+Family Safety. Additionally, parents can also override a block.
+Please see https://www.microsoft.com/en-us/microsoft-365/family-safety for details on Family Safety. 
+
+# Examples
+## WinRT and .NET   
+```c#
+void CreateEnvrionmentWithOption()
+{
+    // If parents set filtering rules to only allowed sites. App developers can use
+    // FamilySafetyAllowedUris to show app content and still honor general filter settings set
+    // by parents.
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.IsFamilySafetyEnabled = true;
+    // appassets.example is used as an example app content. All subdomain are added to the allow list
+    options.FamilySafetyAllowedUris.Add("https://appassets.example/*");
+    auto environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+}
+```
+## Win32 C++
+```cpp
+HRESULT InitializeWebView()
+{
+    // If parents set filtering rules to only allowed sites. App developers can use
+    // FamilySafetyAllowedUris to show app content and still honor general filter settings set
+    // by parents.
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+    options->put_IsFamilySafetyEnabled(TRUE);
+    // appassets.example is used as an example app content. All subdomain are added to the allow list
+    const WCHAR* allowedUris[1] = {L"https://appassets.example/*"};
+    options->SetFamilySafetyAllowedUris(1, allowedUris);
+    HRESULT hr = CreateCoreWebView2EnvironmentWithOptions("", nullptr, options.Get(),
+    Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
+        this, &AppWindow::OnCreateEnvironmentCompleted)
+        .Get());
+}
+```
+
+# API Details    
+```
+interface ICoreWebView2EnvironmentOptions3;
+
+/// Additional options used to create WebView2 Environment.
+[uuid(D0965AC5-11EB-4A49-AA1A-C8E9898F80AF), object, pointer_default(unique)]
+interface ICoreWebView2EnvironmentOptions3 : IUnknown {
+  /// `IsFamilySafetyEnabled` property is to enable/disable family safety feature.
+  /// It is `FALSE` by default.
+  /// When `IsFamilySafetyEnabled` is `TRUE` WebView2 provides the same Family Safety 
+  /// functionality as the Edge browser. 
+  /// Family Safety is a set of features available on Windows for managing children Internet 
+  /// priviliges. Microsoft account may be linked to have a family relationship, with adults 
```

Comment:
```suggestion
  /// privileges. Microsoft account may be linked to have a family relationship, with adults 
```
Overall, this comment block could do with an editing pass.

Reply:
Please fix typo.
Please revise paragraph or ask for PM to help rephrase.